lib/ansible/executor/task_executor.py

```diff
@@ -269,6 +270,8 @@ def _run_loop(self, items):
             self._rslt_q.put(TaskResult(self._host.name, self._task._uuid, res), block=False)
             results.append(res)
             del task_vars[loop_var]
+            if loop_pause:
```

Comment:
it is inside the if from 237, which ensures that loop_control is defined and has a value